docs/en/guide/deploy.md

```diff
@@ -294,6 +294,12 @@ You can deploy your VitePress website on [Kinsta](https://kinsta.com/static-site
 
 You can deploy your VitePress project to [Stormkit](https://www.stormkit.io) by following these [instructions](https://stormkit.io/blog/how-to-deploy-vitepress).
 
+### Azion
+
+[Azion](https://www.azion.com/en/) is a web platform that allows you to deploy your VitePress site globally. It provides a efficient way to host your website with low latency and high availability.
+
+You can follow the [this guide](https://www.azion.com/pt-br/documentacao/produtos/guias/vitepress-typescript-boilerplate/) to set it up.
```

Comment:
The linked docs are in Portuguese.

English version would be `https://www.azion.com/en/documentation/products/guides/vitepress-typescript-boilerplate/`

Also, `follow the [this guide]`?

Reply:
Thanks for the feedback, just updated with correct link and text.